### PR TITLE
Add Additional working, location, and jurisdiction preferences to custom report download.

### PR DESCRIPTION
**Pull request by tomlovesgithub**
Ranked, multiple and single choice questions formatted for download and display on the custom report

### CODE REVIEW

File: nodeScripts/customReports.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+const config = require('./shared/config');
+const { firebase, app, db } = require('./shared/admin.js');
+const { customReport } = require('../actions/exercises/customReport')(config, firebase, db, auth);
```
Comment (HalcyonJAC) [inline nodeScripts/customReports.js:4]: `auth` is not defined here. Maybe it should be imported from './shared/admin.js'.

File: nodeScripts/customReports.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const main = async () => {
+  await customReport({
+    name: params.name,
```
Comment (HalcyonJAC) [inline nodeScripts/customReports.js:8]: `params` is not defined here.

**Comment by andrew-isaac** [comment]
@tomlovesgithub  - Do you have an update on this as Teams are asking again for this information to use in their exercises

**Review by HalcyonJAC — APPROVED** [review]
LGTM 👍

**Pull request merged**